src/Stateless/Reflection/ActionInfo.cs

```diff
@@ -34,4 +37,4 @@ public ActionInfo(InvocationInfo method, string fromTrigger)
             FromTrigger = fromTrigger;
         }
     }
-}
+}
```

Comment:
Newline was removed

Reply:
By the way, I'm not sure if this comment is a request? ie do you need the newline to be put back in?

Reply:
Yes, I think it's a request to put the newline back in. There are several existing files that are missing the newline at the end, and we should probably plan to address those too at some point in the future.

Reply:
No worries. I've done that now.